### PR TITLE
DOCS-1266 cleanup policy update

### PR DESCRIPTION
**Pull request by JoyceFee1**
This PR clarifies  log.cleanup.policy impact on other topics in cluster.

@guozhangwang 

*Summary of testing strategy (including rationale)
for the feature or bug fix. Unit and/or integration
tests are expected for any behaviour change and
system tests should be considered for larger changes.*

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by JoyceFee1** [comment]
@joel-hamill  Please review - formatting errors.

File: docs/design.html [hunk 1]
```diff
@@ -523,9 +523,13 @@ <h4><a id="design_compactiondetails" href="#design_compactiondetails">Log Compac
...
-    To enable log cleaning on a particular topic you can add the log-specific property
-    <pre class="brush: text;">  log.cleanup.policy=compact</pre>
-    This can be done either at topic creation time or using the alter topic command.
+    To enable log cleaning on a particular topic, add the log-specific property
+    <pre class="brush: text;">log.cleanup.policy=compact</pre>.
+    
+    The <code>log.cleanup.policy</code> property is a broker configuration setting defined 
+    in the broker's <code>server.properties</code> file; it affects all of the topics 
+    in the cluster that do not have a configuration override in place as documented 
+    in <a href="/documentation.html#brokerconfigs">.
```
Comment (joel-hamill) [inline docs/design.html]: a href must have closing tag, e.g.

```
in <a href="/documentation.html#brokerconfigs">yourlinktext</a>. 
```

File: docs/design.html [hunk 2]
```diff
@@ -523,9 +523,13 @@ <h4><a id="design_compactiondetails" href="#design_compactiondetails">Log Compac
     <h4><a id="design_compactionconfig" href="#design_compactionconfig">Configuring The Log Cleaner</a></h4>
 
     The log cleaner is enabled by default. This will start the pool of cleaner threads.
-    To enable log cleaning on a particular topic you can add the log-specific property
-    <pre class="brush: text;">  log.cleanup.policy=compact</pre>
-    This can be done either at topic creation time or using the alter topic command.
+    To enable log cleaning on a particular topic, add the log-specific property
+    <pre class="brush: text;">log.cleanup.policy=compact</pre>.
```
Comment (joel-hamill) [inline docs/design.html]: remove the period after `</pre>` (`<pre>` puts the string on line by itself); or change to `<code>` (`<code>` puts string inline).

**Review by joel-hamill — CHANGES_REQUESTED** [review]
Some broken syntax

**Review by joel-hamill — APPROVED** [review]
LGTM

**Comment by joel-hamill** [comment]
ping @guozhangwang

**Pull request merged**